src/generator.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const generateRandomNumber = () => {
+  const random = Math.random() * 10;
```

Comment:
you could also cover the aspect of duplicate numbers
so that that are no 2222 or 1343 etc.